api/Document.json

```diff
@@ -1528,7 +1528,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "9"
```

Comment:
This surprised me, but indeed it seems like `document.createAttributeNS` isn't in IE8. The documents we're testing in are HTMLDocument instances, and it's possible `createAttributeNS` was on XMLDocument, but IE8 actually doesn't seem to have XMLDocuemnt.

https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/ff975561(v=vs.85) seems to support this change.